### DISCUSSION TITLE
How do I hide the labels for the x-axis

### DISCUSSION THREAD

**Discussion by plsmarann** [patrykandpatrick/vico#758] [Questions]
### Question

Using line chart , I need to hide the labels for the x-axis , keeping the spacing specified in the rememberBottomAxis(
                itemPlacer = AxisItemPlacer.Horizontal.default(
                    offset = 0,
                    spacing = xAxisSpacing
                )

### Vico version(s)

1.14.0

**Comment by Gowsky** [comment]
Hello, `rememberBottomAxis` has a `label` parameter that you need to set to `null`.
